tests/tools/test_dist_part.py

```diff
@@ -136,11 +136,18 @@ def test_data(sub_dir, feat, expected_data, expected_shape):
                 test_data(sub_dir, feat, data, g.num_edges(c_etype) // num_chunks)
 
 
-def _test_pipeline(num_chunks, num_parts, graph_formats=None):
+@pytest.mark.parametrize("num_chunks", [1, 2, 3, 4, 8])
```

Comment:
are these `parametrize` required here?

Reply:
Do you mean, test_pipeline_basics function is the only function which be exposed to the user, _test_pipeline function is internal function. So there is no need to parametrize this function. 

I agree with you, _test_pipeline function, need not be parametrized. I will take it away !!!